azerothcore/azerothcore-wotlk
# fix(Core/Scripts): ScriptEffect Frog Kiss

## Changes Proposed:

This PR proposes changes to:

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/6174

## SOURCE:

The changes have been validated through:

https://wowgaming.altervista.org/aowow/?spell=62536
https://www.wowhead.com/quest=13666/a-blade-fit-for-a-champion#comments

## Tests Performed:

This PR has been:


## How to Test the Changes:



.quest add 13666 for alliance, .quest add 13673 for horde
.go xyz 3812.86 -4357.22 182.88 571

Use item then /kiss a frog, buff should disappear
When Maiden of Ashwood spawns, talk to her, she should have only 1 gossip option

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
